lntest/itest/assertions.go

```diff
@@ -277,51 +276,44 @@ func assertChannelClosed(ctx context.Context, t *harnessTest,
 	closeUpdates lnrpc.Lightning_CloseChannelClient) *chainhash.Hash {
 
 	txid, err := lnrpc.GetChanPointFundingTxid(fundingChanPoint)
-	if err != nil {
-		t.Fatalf("unable to get txid: %v", err)
-	}
+	require.NoError(t.t, err, "unable to get txid")
 	chanPointStr := fmt.Sprintf("%v:%v", txid, fundingChanPoint.OutputIndex)
 
 	// If the channel appears in list channels, ensure that its state
 	// contains ChanStatusCoopBroadcasted.
 	ctxt, _ := context.WithTimeout(ctx, defaultTimeout)
 	listChansRequest := &lnrpc.ListChannelsRequest{}
 	listChansResp, err := node.ListChannels(ctxt, listChansRequest)
-	if err != nil {
-		t.Fatalf("unable to query for list channels: %v", err)
-	}
+	require.NoError(t.t, err, "unable to query for list channels")
+
 	for _, channel := range listChansResp.Channels {
 		// Skip other channels.
 		if channel.ChannelPoint != chanPointStr {
 			continue
 		}
 
 		// Assert that the channel is in coop broadcasted.
-		if !strings.Contains(channel.ChanStatusFlags,
-			channeldb.ChanStatusCoopBroadcasted.String()) {
-			t.Fatalf("channel not coop broadcasted, "+
-				"got: %v", channel.ChanStatusFlags)
-		}
+		require.Contains(
+			t.t, channeldb.ChanStatusCoopBroadcasted.String(),
```

Comment:
I think the arguments need to be swapped here.

Reply:
Fixed!